packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -125,7 +125,7 @@ export interface Field<T = any, V = Vector<T>> {
   /**
    * Convert text to the field value
    */
-  parse?: (value: any) => T;
+  // parse?: (value: any) => T;
```

Comment:
@mckn we should likely deprecate first :shrug: and then remove

Given that the function is never called internally, I don't know if there is any benefit to keeping it around though.

Reply:
yea, not sure the point of deprecating it (over removing it). Won't cause any runtime issues anyway